components/supported/SupportedDefi.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div id="defi" class="py-12">
+    <heading>Dedicated DeFi support for</heading>
+    <div class="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 gap-4 mt-8">
```

Comment:
Can we do the same thing as with the rest of the website?
Create a single class where all these are applied and use that class in the code